tests/contest/runtimetest/src/tests.rs

```diff
@@ -550,6 +551,36 @@ pub fn test_io_priority_class(spec: &Spec, io_priority_class: IOPriorityClass) {
     }
 }
 
+pub fn validate_process(spec: &Spec) {
+    let process = spec.process().as_ref().unwrap();
+
+    if process.cwd().ne(&getcwd().unwrap()) {
```

Comment:
I think you should be able to directly use `!=` here. Also, as we are using the process.cwd and getcwd multiple times (in comparison and in the `eprintln` later, extract them as variables.

Reply:
I fixed this.
https://github.com/youki-dev/youki/pull/2968/commits/66a404ed709b82fe3618745f09dd3be2c0fde715